content/api-reference/queries/search-activities.mdx

```diff
@@ -70,3 +74,21 @@ When specifying multiple filters, the resulting criteria is the conjuction of al
   }
 }
 ```
+
+### Pagination and sorting
+
+Pagination and sorting are optional, **though we recommend not implementing this query without pagination**. Not using pagination means that the API will return all activities at once which might cause performance issues.
```

Comment:
I'm a bit hesitant to phrase it this way. I'd rather leave out they are optional and just say that providing pagination and sorting inputs are preferred. We still support queries without these inputs but this will be deprecated in the future. (this makes it clear that they should use the inputs, or face breaking changes going forward)